setup.cfg

```diff
@@ -1,15 +1,3 @@
-[tool:pytest]
-python_files = test*.py
-# Note: When updating pytest tweaks, make sure to update getsentry
-# When updating the traceback format, make sure to update .github/pytest.json
-addopts = --tb=short -p no:doctest -p no:warnings
-norecursedirs = bin dist docs htmlcov script hooks node_modules .* {args}
-looponfailroots = src tests
```

Comment:
what about these options?

Reply:
norecursedirs: Our pytest invocations already specify filepaths. Usually you specify these if you're just invoking `pytest` straight from gitroot.

looponfailroots: undocumented and as far as I can tell, isn't useful

Reply:
edit: looponfailroots is part of pytest-xdist

Reply:
I added it back.

Reply:
Not sure if anyone manually runs just `pytest`,  but would not having `norecursedirs` make things a lot slower for them?